tools/actions/merge.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+# Requires [gh](https://cli.github.com/), [jq](https://jqlang.github.io), git, and grep. Also awk if you pass a URL.
+
+# This script can be used to "purple-merge" PRs that are supposed to land as a single commit, using the "Squash and Merge" feature of GitHub.
+# To land a PR with this tool:
+# 1. Run `git node land <pr-id-or-url> --fixupAll`
+# 2. Copy the hash of the commit at the top of the PR branch.
+# 3. Run `tools/actions/merge.sh <pr-id-or-url> <commit-hash>`.
+
+set -xe
+
+pr=$1
+commit_head=$2
+shift 2 || (echo "Expected two arguments" && exit 1)
+
+commit_queue_failed() {
+  pr=$1
+
+  echo "Failed to merge $pr"
+  cat output
```

Comment:
```suggestion
  cat output || cat output.json
```

Reply:
I'm not sure this is necessary, `output.json` is written once, and we `cat` it immediately, I don't think there would be any value in showing it again. Also, IIUC it's never going to run because `output` would have been created before-hand.

Reply:
Agreed. The only case it would run is when `output` couldn't be created for whatever reason; but even in that case it duplicates previous `cat`.
For the formal correctness of error handling function (to make sure we actually end with the explicit `exit 1` rather than being cut by `set -e`), we can just swallow the potential error (`cat output || true`).

Reply:
Using the `commit_queue_failed` function was silly, I removed it, PTAL

Reply:
Latest comments were about the version w/o `commit_queue_failed`; but these are safe-to-ignore nits.